src/main/resources/fabric.mod.json

```diff
@@ -33,7 +33,7 @@
   },
   "depends": {
     "fabricloader": ">=0.8.8",
-    "minecraft": ["1.16.1", "1.16.2"],
+    "minecraft": ["1.16.x"],
```

Comment:
This would include snapshots too (as it matches anything resembling 1.16). Whilst some snapshots would indeed work, such as the 1.16.3 release candidate, I think we should be sticking to just the concrete versions.

Reply:
Agreed, I can revert this and include 1.16.3, one sec

Reply:
Fixed :+1: